src/js/store/projects/actions.ts

```diff
@@ -20,6 +21,27 @@ export const updateProject = (payload: Project): ProjectUpdated => ({
   payload,
 });
 
+export const setUsersOnProject = (payload: Project): ThunkResult => async (
+  dispatch,
+) => {
+  try {
+    const response = await apiFetch({
+      url: window.api_urls.project_detail(payload.id),
+      dispatch,
+      opts: {
+        method: 'PUT',
+        body: JSON.stringify(payload),
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      },
+    });
+    dispatch({ type: 'PROJECT_UPDATE', payload: response });
+  } catch (error) {
+    console.warn(error);
+  }
+};
```

Comment:
I'll note that there's really nothing specific about setting users in this action -- it just accepts an object, and does a `PUT` to a URL, then fires an action. I'm okay with it here (though it could be renamed to `updateProject` or something), but I think eventually we'll want to abstract it into a generic `updateObject` function similar to the other actions in `store/actions.ts`.